ReStyle-Frontend/src/app/app.module.ts

```diff
@@ -9,6 +17,41 @@ import { ItemCardStackComponent } from './components/item-card-stack/item-card-s
 import { HeaderComponent } from './components/header/header.component';
 import { FooterComponent } from './components/footer/footer.component';
 import { TradePageComponent } from './components/trade-page/trade-page.component';
+import { HomePageComponent } from './components/home-page/home-page.component';
+import { LoginCardComponent } from './components/login-card/login-card.component';
+import { LoginComponent } from './components/login/login.component';
+
+
+const firebaseUiAuthConfig: firebaseui.auth.Config = {
+  signInFlow: 'popup',
+  signInOptions: [
+    firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+    {
+      scopes: [
+        'public_profile',
+        'email',
+        'user_likes',
+        'user_friends'
+      ],
+      customParameters: {
+        auth_type: 'reauthenticate'
+      },
+      provider: firebase.auth.FacebookAuthProvider.PROVIDER_ID
+    },
+    firebase.auth.TwitterAuthProvider.PROVIDER_ID,
+    firebase.auth.GithubAuthProvider.PROVIDER_ID,
+    {
+      requireDisplayName: false,
+      provider: firebase.auth.EmailAuthProvider.PROVIDER_ID
+    },
+    firebase.auth.PhoneAuthProvider.PROVIDER_ID,
+    firebaseui.auth.AnonymousAuthProvider.PROVIDER_ID
+  ],
+  tosUrl: '<your-tos-link>',
+  privacyPolicyUrl: '<your-privacyPolicyUrl-link>',
```

Comment:
Also Privacy Policy Url (I forgot to add them)
- Need dummy page for Privacy Policy as well